yml/cdb_latiss.yaml

```diff
@@ -760,11 +762,13 @@ tables:
     description: Were any control system components simulated?
   - name: vignette
     "@id": "#visit1.vignette"
-    datatype: int
-    description: "Instrument blocked from the sky: Unknown = 0, No = 1, Partially = 2, Fully = 3."
+    datatype: string
+    length: 10
+    description: "Instrument blocked from the sky: UNKNOWN, NO, PARTIALLY, FULLY."
   - name: vignette_min
     "@id": "#visit1.vignette_min"
-    datatype: int
+    datatype: string
+    length: 10
     description: Minimum value of vignette during the exposure.
```

Comment:
What is the definition of "minimum" after this change?

Reply:
Closest to the beginning of the list.  Casting around for better phrasing...

Reply:
How about "Lowest amount of instrument vignetting detected during the exposure."

Reply:
Along with the same enum values again.

Reply:
So "unknown" wins, on purpose?

Reply:
Yes.